Assets/Scripts/Internal/DaggerfallBillboard.cs

```diff
@@ -439,6 +439,27 @@ public void OnDrawGizmos()
                 }
             }
         }
+        
+        /// <summary>
+        /// Draws a circle at the bottom of the bilboard to make it easier to judge the size regardless of rotation.
+        /// </summary>
+        public void OnDrawGizmosSelected()
+        {
+            if (!Application.isPlaying)
+            {
+                UnityEditor.SceneView sceneView = GetActiveSceneView();
+                if (sceneView)
+                {
+                    float radius = 0.0f;
+                    Vector3 offset = Vector3.zero;
+
+                    radius = (summary.Size.x / 2);
+                    offset.y = (summary.Size.y / 2);
+
+                    Handles.DrawWireDisc(transform.position - offset, Vector3.up, radius);
```

Comment:
consider more concise form:
```
var sizeHalf = summary.Size * 0.5f;
UnityEditor.Handles.DrawWireDisc(transform.position - new Vector3(0, sizeHalf.y, 0), Vector3.up, sizeHalf.x);
```

Reply:
Thanks for the comment, to be perfectly honest, this was basically what I started with but I started second guessing myself so I tried to make it match the existing OnDrawGizmos function to try to lessen the likelihood of it getting rejected.  I'll fix it up in the morning, Though at this point me rewriting it almost seems like a formality.

Reply:
Updated.